Documentation/guides/quick_start.md

```diff
@@ -103,44 +90,25 @@ The most broadly applicable advice is to go to [Econ-ARK](https://econ-ark.org)
 
 ## Making changes to HARK
```

Comment:
Arguably this doesn't belong in the quick-start guide -- perhaps just redirect to the contributing guide?